changes/202202151235.feature

```diff
@@ -0,0 +1 @@
+Added missing mocks
```

Comment:
Would this be a `.feature`? Wouldn't adding tests come under:
> Changes to the repository that do not impact functionality
Which would make this a `.misc`

Or maybe a `.bug` if the mocks were meant to be there?

Reply:
in most cases, this would be true but in this case the Mocks are exported so that they can be used in projects importing this library. For those projects, new mocks are hence new "features"

Reply:
Ah okay